frontend/src/metabase-types/api/permissions.ts

```diff
@@ -72,6 +72,7 @@ export type NativePermissions =
   | DataPermissionValue.QUERY_BUILDER_AND_NATIVE
   | DataPermissionValue.QUERY_BUILDER
   | DataPermissionValue.NO
+  | Record<SchemaName, Record<DatabaseId, DataPermissionValue>>
```

Comment:
We cannot pass the schema object (e.g. `{public: ...}` to NativePermissions even when it is supported by the API, we update the types to allow this.